README.md

```diff
@@ -1888,6 +1907,74 @@ Profile name of the Encapsulation. Valid values are String only.
 ##### `dot1q_map`
 The encapsulation profile dot1q vlan-to-vni mapping. Valid values are an array of [vlans, vnis] pairs.
 
+--
+### Type: cisco_evpn_multisite
+
+Manages Cisco Ethernet Virtual Private Network (EVPN) Multisite configurations of a Cisco device.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I7(2)        | 1.9.0                  |
+| N3k      | not applicable     | not applicable         |
+| N5k      | not applicable     | not applicable         |
+| N6k      | not applicable     | not applicable         |
+| N7k      | not applicable     | not applicable         |
+| N9k-F    | not applicable     | not applicable         |
+| N3k-F    | not applicable     | not applicable         |
+
+
+#### <a name="cisco_evpn_multisite-caveats">Caveats</a>
+
+The `cisco_evpn_multisite` is only supported on N9K-EX and N9K-FX devices. For eg: N9K-C93180YC-EX.
+
+#### Parameters
+
+##### `ensure`
+Determines whether or not the config should be present on the device. Valid values are 'present' and 'absent'. Default value is 'present'.
+
+##### `multisite`
+The EVPN Multisite identifier. Valid values are Integer.
+
+#### Properties
+
+##### `delay_restore`
+
+Delay restore time in seconds. Valid values are Integer or default.
+
+--
+### Type: cisco_evpn_stormcontrol
+
+Manages Cisco Ethernet Virtual Private Network (EVPN) stormcontrol configurations of a Cisco device.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I7(2)        | 1.9.0                  |
+| N3k      | not applicable     | not applicable         |
+| N5k      | not applicable     | not applicable         |
+| N6k      | not applicable     | not applicable         |
+| N7k      | not applicable     | not applicable         |
+| N9k-F    | not applicable     | not applicable         |
+| N3k-F    | not applicable     | not applicable         |
+
+
+#### <a name="cisco_evpn_stormcontrol-caveats">Caveats</a>
+
+The `cisco_evpn_stormcontrol` is only supported on N9K-EX and N9K-FX devices. For eg: N9K-C93180YC-EX.
+
+#### Parameters
+
+##### `ensure`
+Determines whether or not the config should be present on the device. Valid values are 'present' and 'absent'. Default value is 'present'.
+
+##### `packet_type`
+The packet type to apply stormcontol on. Valid values are 'unicast', 'multicast' or 'broadcast'.
+
+#### Properties
+
+##### `level`

```

Comment:
You use `level` to initialize the evpn_stormcontrol object.  It should be a parameter right?

Reply:
As discussed, I've left `level` as a property as it is not required to define the resource. I have made it a requires property if 'ensure => present'